tests/net/socket/getaddrinfo/testcase.yaml

```diff
@@ -0,0 +1,7 @@
+tests:
+  test:
+    extra_configs:
+      - CONFIG_NET_TEST=y
```

Comment:
I suggest putting these two extra config values into prj.conf file so that they are automatically applied. Now the settings are used only for sanitycheck script.